src/transformers/models/parakeet/modular_parakeet.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, config: ParakeetEncoderConfig, layer_idx: int):
         super().__init__(config, layer_idx=layer_idx)
         self.is_causal = False
         # W_{k,R} projection
-        self.relative_k_proj = nn.Linear(config.hidden_size, config.num_attention_heads * self.head_dim, bias=False)
+        self.relative_k_proj = nn.Linear(config.hidden_size, config.num_attention_heads * self.head_dim, bias=config.attention_bias)
```

Comment:
Interesting that `bias` was hardcoded to False...

Can you check if this breaks Parakeet CTC? It has `attention_bias=True` (see [here](https://huggingface.co/nvidia/parakeet-ctc-1.1b/blob/main/config.json#L10)) but would have been hardcoding to False before.